src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/live_event.py

```diff
@@ -4,14 +4,21 @@
 # --------------------------------------------------------------------------------------------
 
 
-def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location):
+def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location,
+           auto_start=False, encoding_type=None, preset_name=None, tags=None, description=None):
     from azure.mgmt.media.models import LiveEventInputProtocol
     from azure.mgmt.media.models import LiveEventInput
     from azure.mgmt.media.models import LiveEvent
+    from azure.mgmt.media.models import LiveEventEncodingType
+    from azure.mgmt.media.models import LiveEventEncoding
 
     protocol = LiveEventInputProtocol(streaming_protocol)
     liveeventinput = LiveEventInput(streaming_protocol=protocol)
-    live_event = LiveEvent(input=liveeventinput, location=location)
+    encodingtypeenum = LiveEventEncodingType(encoding_type)
+    liveeventencoding = LiveEventEncoding(encoding_type=encodingtypeenum, preset_name=preset_name)
+
+    live_event = LiveEvent(input=liveeventinput, location=location, encoding=liveeventencoding)
```

Comment:
Variable names should be use _ (e.g instead of liveventencoding we should use live_event_encoding (apply the change where needed)